src/ru/pro/sky/Main.java

```diff
@@ -0,0 +1,126 @@
+package ru.pro.sky;
+
+public class Main {
+    //Базовая сложность
+    private static Employee[] employees = new Employee[10];
+
+    public static void printArray (Employee [] employees) {
+        for (int i = 0; i < employees.length ; i++) {
+            if (employees[i] !=null ){
+                System.out.println(employees[i]);
+            }else {
+                break;
+            }
+        }
+    }
+    public static void calculateSalarySum(Employee[] employees) {
+        double total = 0;
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i] != null)
+                total += employees[i].getSalary();
+            if (employees[i] == null) {
+                break;
+            }
```

Comment:
условия взаимоисключающие
```suggestion
            if (employees[i] != null) {
                total += employees[i].getSalary();
           } else {
                break;
            }
```